scss/_nav.scss

```diff
@@ -127,7 +127,7 @@
     // stylelint-disable-next-line function-disallowed-list
     border-width: 0 0 calc(var(--#{$prefix}nav-tabs-border-width) * 4);
 
-    &:hover,
+    &:not(.active):hover,
```

Comment:
IDK if this rule is useful.

Reply:
It was indeed useful for the previous version of the issue.
I had to revamp it a bit for the updated version.